src/2sv/smartphone/VerifyQrCode.vue

```diff
@@ -1,38 +1,39 @@
 <template>
   <ProfileWizard>
     <BasePage>
-      <template v-slot:header>
-        {{ $vuetify.lang.t('$vuetify.2sv.smartphone.verifyQrCode.header') }}
+      <template #header>
+        {{ $t('2sv.smartphone.verifyQrCode.header') }}
       </template>
 
-      <v-row align="center" class="px-5">
-        <v-form @submit.prevent="verify" ref="form" class="pl-2 d-flex flex-column align-center">
-          <p>{{ $vuetify.lang.t('$vuetify.2sv.smartphone.verifyQrCode.info') }}</p>
+      <v-row class="px-5">
+        <v-form ref="form" class="pl-2 d-flex flex-column align-center" @submit.prevent="verify">
+          <p>{{ $t('2sv.smartphone.verifyQrCode.info') }}</p>
 
           <BaseTextField
-            type="text"
-            :label="$vuetify.lang.t('$vuetify.2sv.smartphone.verifyQrCode.codeInput')"
             v-model="code"
+            width="100%"
```

Comment:
Is the addition of `width="100%"` related to the upgrade, or is it just a general improvement?

Reply:
For the upgrade. For some reason the field was very small so I had to do this to fill it's parent. Not sure what changed that required this.